zeton/templates/index_child.html

```diff
@@ -49,39 +49,39 @@ <h2 class="text-center">Wykorzystaj punkty</h2>
 
             {% if role == 'caregiver' %}
             <form action="/api/child/{{ child.id }}/points/use?return_url=/child/{{ child.id }}" method="post">
-            {% else %}
-            <form action="/api/child/{{ child.id }}/points/use" method="post">
-            {% endif %}
-                <div class="form-group button">
-                    Liczba punktów: <input class="form-control" type="number" name="points" required>
+                {% else %}
+                <form action="/api/child/{{ child.id }}/points/use" method="post">
+                    {% endif %}
+                    <div class="form-group button">
+                        Liczba punktów: <input class="form-control" type="number" name="points" required>
 
-                    <input class="btn btn-info" type="submit" name="wykorzystaj punkty" value="Wykorzystaj punkty" />
-                </div>
-                <h4 class="text-center">NAGRODY</h4>
-            </form>
+                        <input class="btn btn-info" type="submit" name="wykorzystaj punkty" value="Wykorzystaj punkty"/>
+                    </div>
+                    <h4 class="text-center">NAGRODY</h4>
+                </form>
```

Comment:
po linii 61 trzeba zamknąć jeszcze form z linii 51

Reply:
To nie są zmiany z mojego brancha. Ale super, że to zauważyłaś :) Poprawiłam to.